ftp-ballerina/tests/resources/private.key

```diff
@@ -0,0 +1 @@
3mhzV15kUDMipOTliGEZVJpuLl2dBNKRvjfvSiZsM2DkWiMQnHtM1O5VBgjjzaFM
LfidWhBcjdgWkmK3AxssoohFTMMeTZFudV+HssE2a951wBlHwOi0itxbgwL+ErM/
Eyne7umNdJG6FUhbcW6kYPWNbQRcWpec0qBKAheTKxmNKuHdE7oB7q4UiVkvXBvp
N+qlfMlHJXhS343He3TQtZ2mV4zhDwHuW4nVwtxgQBnxr82R2LcgEYYq36EtZhsi
kX/gVBvhgV4FVzA/VLJTxOtihNHE9W3Ojp78WMrLpheec22tCMhQ6S9hjp2H+ubA
mlTcEJLMwaGBjul/B1zSurYZLmI66iSR2qkTcj6nyBypdBrQ7RVIq0iMoQKYV79f
imJ887UEVuZ5/k5sGsGlqyFR3VJzudFik1kYe8mc12QMpf2FK3rI7e+22OykEogM
ZAaoP2AKzagsUs4cdtpjBgm1SUx3+wIDAQABAoICADAutZnjFcAyr8pEM63Xh+jk
Xx6w6d/K4qNOJVyLHO1sA1Lj5AcxKNf6opavyHVt/f1wtius2g9F49vGqTIxjpSO
b2PSljQxuSoUewJ8BKCdKQQVQjbxpvpukkgZO3xTplUc56so/EEvyZiTIA5boYQy
NOSDHVkDlQdJ3AQqNSceGZvJLO4KgKtq0IfppuEqjmDNHr2/Z92GtPDlhWap0Wfc
tYME8IICbN9vl2ptAYHYfJs6ulbaOzh8RA5JPjFNYowk8t5qC9qjfo5Z2E1/tu2Y
riWikiEGKL/MgmvdSNIVd1NTRmMbZ84wIlzBbtKPiwrAerYI07UWDe3aWjPJabEE
flaEC0qvo7KhET/qFfsZcx/noXhi65WCQq4i20ex65fIyFTPE4BTdCb2d5Es2r5h
5qA88vgis2rTU315eUbSZPGDx58nAvC/Sd702zukx5Dvlt/JNE4AenLXEhsuO4vk
tSkcf1UMNbq2QMWzd05htr6UVeq/dMhs0TaCvOhqjSGe8CKouZWSY9seB6I9WUc0
aCZD4Hgyafytjap5ytR5ZMNIAKLrqnewDFq/upXd/OeWukDF7WhDnrV1e2TZ7Ahv
5gKd9RxKZn4VLketuSxXNC3dL+VBGQotqYgqOAQA9hsKwDP+19r3l7IqKpKcTAmg
Mih6YoFz6zOWqlmgM4bhAoIBAQDj5LZvw5LdPSL9cEsDdjXEsJ4Ss6kb6He4xhSL
Jsr/fUM8YjqtiabTeAmNPT/gI+ku+hhkjTGz2lG+1qNkKPs2l2g5EUs7nbtnhGEm
REAKVaYf6vDePLsDzv1/g/f6jBhVbj5iND008HHJazgItFR2CB6HNzSoRTqNbQpu
VJCrD4VUinfDw865cbp7mYQzUTxcs0kfpckdpgpvWTT5J1qThMEuI+J2hnJXpuUo
/nHSmnxfmEzPAVw48z7W+2BU02k9Znjg3ejyCU4AsDukv2nCWxgstBXpY7EtTH8o
wAgGYyPsobrIYlfc+xkSzj0Jbj/9u44PcBx9WVU2ZPok6OfxAoIBAQDIZQFp76aJ
BdbhNy1HHyYPBa3oeL2WR4N95K7ZtGjwDEzI4o5FnRFoOQxGQb5Y8sf6U1StmnqY
VQoj0slvHsNUB7e+sT/ZdmK/0MjqdgirWYL7Bi99dumSfoq4KJBmE10L7jBdRExa
HdnEm83r8EozZSmoNNAovZEwXG7LVDd2TNSarbZWQ9v+sTWWZdZWuASegB9bOS5M
1kfCOY8pe2dqHcJJgu+yA2Tar6m1z11fxcrA5QBIvS+q8GJMvGRg+BqwPFshO4wW
580ewhBr515dNmFKK+UgW304C5SM7enF05k/yxpgk9/ksiMPUZnlvaRH5L2q8pM5
qiJsCpmybCqrAoIBADsdUNWc/bO18CWdw1eaikgDKH3hbLR4kNW1TA7seSvc6lv3
fxEU69lfAdtJUBBsQEiXR0yN856buyittKR+nOuEStzCx6knOT2Y8Tpj6djEbttI
rkJ3rmtGfitdqC2tuYIvnq8KrfXViS6eSWK4pFAZuxGIGHitvIqDBnmoJf65f/R2
FqsJV85y2dTF9az6HZZOS2GSN6EhOQO5OCyJMM2jWhkF5fCo33AtMrR3jD79I9YI
Q6ROzgVX060gd9SJInty4VgzOAHQuBMlo9MQc/aeep7d9/EakxMVDMcdSPefL+pp
6XGhUPlZiR9/sAWKO78tr6QBiOJbEeeSOfDpGYECggEAcNevvB+e/3myTo/rhykb
HUpta4LR/1W1zaaAb0cP8Fx8wIIS61BPBQlayBUtQpkz0topzLg4YJ9CXnnLxC7m
fuL5rP/SnHn1kV3YRW7B4vs8rm4FP/k9/agUJS9yYIh21MkDowFanURB5Fx30Gw2
3EoS8w0KSma04y6plQWPVaZd8dvv2RPNRdYz/sTJtXtZ3ob3IT9JuyrOragL3agH
y9GGHi8FGpCesiZIwoBKbCvMyGIrjso5lWBXei57t77+CE90YXpOoHdXIgLVjG10
zbhZgJLHKzlZgb3DoLg385X7rdiIuemLIhTNeUrSBnSLlZBQ3XpLPSV4zK4LblVK
kQKCAQB639C5kdStTNxWpjadkkghsWj8cPci59kacJJ64AVPOxr2SGjWdttaVlsi
qUng2tJtbonpS+3sdWh7bDslFHJuXZ/wSTwK0PR3fBSzickyBfV97YoO+zT1ewoI
w6sd+Lu6xW7/q57QTK99knP0H/SXtE+DQwB0lfq5h2IhpySL1HMK50SqLmGx3TOv
37TvsYbqzfMjYZ/rf0lQdOTfbI1Ed/RkdH3SGj1InNmWf+N4cjlu41+vfiCyB//H
sQ+f8BwquEhRdMCuzBgPG0G0UZFjWeBhTRD9hQGQ8pNcyuHqTfLup04rfXLX88Nm
15kyqqYzZqslTOpXF8xX87VVkPxd
-----END PRIVATE KEY-----

```

Comment:
Shall we make this multiline and add a new line at EOF? The GitHub preview does that formatting for us when we select 'view file' option. But when we get that pull this content, it does not be always formatted. There is only a singe line with a line no. There are few other files with same concern as well. WDYT?

Reply:
As this file is viewed as multiline with both IntelliJ Idea and Ubuntu text editor as discussed offline lets keep this format as it is for the moment.